Moodis/Moodis/Feature/Login/Serializer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moodis.Feature.Login
+{
+    public static class Serializer
+    {
+        public static void Save(string filePath, object objToSerialize)
+        {
+            try
+            {
+                using (Stream stream = File.Open(filePath, FileMode.Create))
+                {
+                    BinaryFormatter bin = new BinaryFormatter();
+                    bin.Serialize(stream, objToSerialize);
+                }
+            }
+            catch (IOException)
```

Comment:
Print out something to console line or add a label to form, so user would know that saving failed

Reply:
still missing in newest commit.